tests/test_persisted_scale_property.py

```diff
@@ -0,0 +1,21 @@
+"Contains a collection of pytest tests that validates the persisted scale properties."
+
+import nidaqmx
+import nidaqmx.system.system
+from nidaqmx.tests.test_read_write import TestDAQmxIOBase
+
+
+class TestPersistedScaleProperty(TestDAQmxIOBase):
+    """Contains a collection of pytest tests that validates the persisted scale properties."""
+
+    def test_bool_property(self):
+        """Test for validating bool attributes in persisted scale."""
+        system = nidaqmx.system.System.local()
+        persisted_scale = system.scales[0]
```

Comment:
Is it fine to take the first scale from the ni max configuration or would it be better to take the scale based on name, so that we could make sure we are not taking any unknown scale from NI max